defaults/main.yml

```diff
@@ -9,4 +9,6 @@ etcd_ca_depot: ""
 etcd_certs_root: "/etc/ssl/etcd"
 local_certs: "certs"
 
-etcd_core_group: ""
\ No newline at end of file
+etcd_core_group: ""
+
+etcd_image_tag: "v3.1.0"
```

Comment:
Its's downloaded from quoy.io by default. I've added an option to update ETCD_IMAGE_URL to a different source.